jdaviz/core/loaders/importers/image/image.vue

```diff
@@ -10,6 +10,16 @@
       :api_hints_enabled="api_hints_enabled"
       hint="Extension to use from the FITS HDUList."
     ></plugin-select>
+    <plugin-dataset-select
+      :items="parent_items"
+      :selected.sync="parent_selected"
+      :show_if_single_entry="false"
+      :multiselect="false"
+      label="Parent Dataset"
+      api_hint="ldr.importer.parent ="
+      :api_hints_enabled="api_hints_enabled"
+      hint="Select a parent dataset to associate the new data with."
```

Comment:
maybe add a little more info about what this means and when it's relevant? we're familiar with the term 'parent' in this context but might be confusing to a new user. i also think this should be hidden when there is no existing data